src/ResourceManager/Common/Commands.Common.Authentication.ResourceManager/AzureRmProfile.cs

```diff
@@ -573,7 +573,8 @@ public bool TryCopyProfile(AzureRmProfile other)
                 TrySetContext(context.Key, context.Value);
             }
 
-            if (other.DefaultContext != null)
+            this.DefaultContextKey = other.DefaultContextKey;
```

Comment:
If I'm reading the code properly, this assignment and changes to the if-statement are not necessary. The statement above this sets the Contexts. If `other.DefaultContextKey` exists in `other.Context`, then it will be assigned in [TrySetDefaultContext](https://github.com/AMoghrabi/azure-powershell/blob/dd953b32d04ff025ec69ff800f3e3506afe12bf8/src/ResourceManager/Common/Commands.Common.Authentication.ResourceManager/AzureRmProfile.cs#L468-L478).